Improvements/average.js

```diff
@@ -0,0 +1,6 @@
+function calculateAverage(numbers) {
+  let sum = numbers.reduce((accumulator, number) => {
```

Comment:
Well done. `sum` is not reassigned. You can declare it with `const`. 
The `reduce` method gives you access to four parameters. Check it out to see how you could probably re-write the function.